Auth0/WebAuth.swift

```diff
@@ -190,6 +190,14 @@ public protocol WebAuth: Trackable, Loggable {
     @available(iOS 11, *)
     func useLegacyAuthentication() -> Self
 
+    /**
+     Change modal presentation style if using `SFSafariViewController`.
+     
+     - returns: the same WebAuth instance to allow method chaining
+     */
+    @available(iOS 11, *)
+    func legacyAuthenticationPresentationStyle(_ style: UIModalPresentationStyle) -> Self
```

Comment:
I'm not convinced that we need a new protocol method, also a UI method doesn't feel right in this protocol.  Perhaps should be combined as a param in the `useLegacyAuthentication(withStyle: ...)` ?

Reply:
I have made the change request, have extended useLegacyAuthentication with a withStyle parameter to accept a custom style or use the previous default value.